python/mxnet/gluon/contrib/estimator/estimator.py

```diff
@@ -392,7 +392,7 @@ def fit(self, train_data,
             for handler in epoch_begin:
                 handler.epoch_begin(estimator_ref)
 
-            for i, batch in enumerate(train_data):
+            for _, batch in enumerate(train_data):
```

Comment:
in most of the cases enumerate makes no sense

Reply:
```suggestion
            for batch in train_data:
```